pc_ble_driver_py/ble_adapter.py

```diff
@@ -335,16 +339,16 @@ def service_discovery(self, conn_handle, uuid=None):
         return BLEGattStatusCode.success
 
     @NordicSemiErrorCheck(expected=BLEGattStatusCode.success)
-    def enable_notification(self, conn_handle, uuid):
+    def enable_notification(self, conn_handle, uuid, attr_handle=None):
         cccd_list = [1, 0]
 
-        handle = self.db_conns[conn_handle].get_cccd_handle(uuid)
-        if handle is None:
+        ccc_handle = self.db_conns[conn_handle].get_cccd_handle(uuid, attr_handle)
```

Comment:
I'd prefer to use cccd instead of ccc, to be more inline with the rest of the code base. (Same for the other occurences).